core/server/data/migration/reset.js

```diff
@@ -12,11 +11,23 @@ var Promise      = require('bluebird'),
  * Deletes all the tables defined in the schema
  * Uses reverse order, which ensures that foreign keys are removed before the parent table
  *
+ * @TODO:
+ * - move to sephiroth
+ * - then deleting migrations table will make sense
+ *
  * @returns {Promise<*>}
  */
 reset = function reset() {
+    var result;
+
     return Promise.mapSeries(schemaTables, function (table) {
         return commands.deleteTable(table);
+    }).then(function (_result) {
+        result = _result;
+
+        return commands.deleteTable('migrations');
```

Comment:
see PR + PR description https://github.com/TryGhost/Ghost/pull/7501
Migration runner adds a collection. 
We don't use settings table for migrations anymore. Migrations !== settings.